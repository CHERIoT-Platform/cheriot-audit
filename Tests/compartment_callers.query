--board inputs/sail.json -j inputs/test-suite.json -q 'data.compartment.compartments_calling("allocator_test")'
