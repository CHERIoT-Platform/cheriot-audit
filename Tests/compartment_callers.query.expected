["allocator_test","test_runner"]
